scripts/db_migrations/v2.18/upload_models_s3.py

```diff
@@ -0,0 +1,66 @@
+import json
+import gridfs
+import sys
+import uuid
+from pymongo import MongoClient
+import boto3
+
+##### Connect to AWS S3 #####
+s3 = boto3.client('s3')
+bucket_name = '3drepo-models'
+
+mongoURL = sys.argv[1]
+mongoPort = sys.argv[2]
+userName = sys.argv[3]
+password = sys.argv[4]
+connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
+##### Enable dry run to not commit to the database #####
+dry_run = True
+
+##### Connect to the Database #####
+def upload(col_prefix, col_insert_prefix, file_type):
+	db = MongoClient(connString)
+	for database in db.database_names():
+		if database != "admin" and database != "local" and database == "kierongibbons":
+			db = MongoClient(connString)[database]
+			print(file_type + ": --database:" + database)
+
+##### Get a model ID #####
+			for settings in db.settings.find():
+				model_id = settings.get('_id')
+				col_name = model_id + col_prefix
+				fs = gridfs.GridFS(db, col_name)
+				fs_model = fs.find()
+				for model in fs_model:
+					filename = model.filename
+					if file_type == "asset files":
+						filename = filename.replace("/", "-")
```

Comment:
- see if you can do this from a buffer ([this](https://stackoverflow.com/questions/45699905/csv-file-upload-from-buffer-to-s3) and [this](https://stackoverflow.com/questions/36568713/boto3-can-we-use-actual-data-buffer-as-parameter-instaed-of-file-name-to-uploa) might help)

- If you really have to, make sure you use a different variable name. (Also don't make it unique to each file)